src/utils/featheryClient/index.ts

```diff
@@ -707,33 +686,15 @@ export default class FeatheryClient extends IntegrationClient {
     };
 
     let prom = null;
-    let stepKey = '';
-    if (eventData.event === 'load') {
-      stepKey = eventData.previous_step_key;
-    } else {
-      stepKey = eventData.step_key;
+    if (eventData.event !== 'load') {
       prom = this.flushCustomFields();
     }
 
     const triggerEvent = () =>
-      this.offlineRequestHandler.runOrSaveRequest(
-        // Ensure events complete before user exits page. Submit and load event of
-        // next step must happen after the previous step is done submitting
-        () =>
-          this.submitQueue.then(() =>
-            this._fetch(url, options, true, true).catch((e) => {
-              if (e instanceof TypeError && navigator.onLine)
-                // Wait 5 seconds since event may have actually been registered
-                // and just needs to be processed. If online, means it's not an
-                // offline error.
-                return new Promise((resolve) => setTimeout(resolve, 5000));
-              throw e;
-            })
-          ),
-        url,
-        options,
-        'registerEvent',
-        stepKey
+      this.submitQueue.then(() =>
+        wrapUnload(() =>
```

Comment:
if you're removing the typerror await handler, saad should be looped in for this. we added it initially for hiscox.